cts/cts-cli.in

```diff
@@ -1309,8 +1309,25 @@ function test_tools() {
     cmd="crm_resource -a -r gr2 --force"
     test_assert $CRM_EX_OK 0
 
+    # Create a shadow CIB based on constraints.xml
+    create_shadow_cib --create
     unset CIB_file
 
+    desc="Set a meta-attribute for primitive and resources colocated with it"
+    cmd="crm_resource -r prim5 --meta --set-parameter=target-role -v Stopped --recursive"
```

Comment:
Do you want to do these with XML output and validate them, instead of text output? I think we talked about wanting to do our `cts-cli` tests in XML (at least) since that's what we really care about.

Reply:
That's a good idea but uncovers another problem, the XML output in this case contains only the exit status, not the attributes that were changed (which the text output has, and is important to verifying the test). I'll leave that for another time ...